fe2-android/app/src/main/java/com/github/dedis/popstellar/model/network/method/message/data/consensus/ConsensusFailure.java

```diff
@@ -11,7 +12,7 @@
   private final String instanceId;
 
   @SerializedName("message_id")
-  private final String messageId;
+  private final MessageID messageId;
```

Comment:
In toString() line 79 : `messageId` should be changed to `messageId.getEncoded()`

Reply:
Nice catch !